File: src/main/java/club/xiaojiawei/hearthstone/entity/CommonEntity.java
package club.xiaojiawei.hearthstone.entity;

import club.xiaojiawei.hearthstone.enums.ZoneEnum;
import lombok.Data;
import lombok.ToString;

/**
 * @author 肖嘉威
 * @date 2022/11/30 12:38
 */
@Data
@ToString(callSuper = true)
public class CommonEntity extends Entity{

    private String entity;

    private ZoneEnum zone;

    private int zonePos;

    private String playerId;

}

File: Hearthstone-Script/src/main/java/club/xiaojiawei/hsscript/bean/log/CommonEntity.kt
package club.xiaojiawei.hsscript.bean.log

import club.xiaojiawei.bean.Entity
import club.xiaojiawei.enums.ZoneEnum

/**
 * @author 肖嘉威
 * @date 2022/11/30 12:38
 */

open class CommonEntity : Entity() {

    var entity: String = ""

    var zone: ZoneEnum? = null

    var zonePos = 0

    var playerId: String = ""

    var logType: String = ""

}
